ml-agents/mlagents/trainers/sac/optimizer_torch.py

```diff
@@ -167,6 +170,17 @@ def _move_to_device(self, device: torch.device) -> None:
         self.target_network.to(device)
         self.value_network.to(device)
 
+    def _split_discrete_continuous(
```

Comment:
Lets make this a utility func

Reply:
I plan to remove this in favor of Tuples

Reply:
Yea this comment was from before the addition of AgentActions. It can be totally removed.